index.js

```diff
@@ -19,9 +40,40 @@ mongoose
     // Before adding any documents to the database, let's delete all previous entries
     return self.connection.dropDatabase();
   })
+
+  //Iteration 2 - Create a recipe
+   .then(() => {
+   Recipe
+    .create(myRecipe)
+    .then(recipe => console.log(recipe.title))
+    .catch(error => console.error('Creating new recipe error:', error))
+  }) 
+  //Iteration 3 - Insert multiple recipes
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
-  })
+     Recipe
+      .insertMany(data)
+      .then(totallistrecipes => totallistrecipes.forEach(recipe => console.log(recipe.title)))
+      .catch(error => console.error('Inserting multiple recipes error:', error))
+    }) 
+  //Iteration 4 - Update recipe
+  .then(() => {
+    Recipe
+    .findOneAndUpdate({title: 'Rigatoni alla Genovese'}, { $set: {duration: '100'}}, {new: true})
```

Comment:
Your findOneAndUpdate already knows that the second key is the one to update once it recognises it, so your $set is unnecessary here ($set usually adds new fields to a document; it can also update, but it is used more with the method .aggregate())